RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHBHE_cfi.py

```diff
@@ -86,3 +86,159 @@
     energyCorrector = cms.PSet()
 )
 
+#####
+
+seedFinderThresholdsByDetector2017 = particleFlowClusterHBHE.seedFinder.thresholdsByDetector
+
+seedFinderThresholdsByDetector2018 = particleFlowClusterHBHE.seedFinder.thresholdsByDetector
+seedFinderThresholdsByDetector2018 = cms.VPSet(
+    cms.PSet(
+        detector = cms.string("HCAL_BARREL1"),
+        depths = cms.vint32(1, 2, 3, 4),
+        seedingThreshold = cms.vdouble(1.0, 1.0, 1.0, 1.0),
+        seedingThresholdPt = cms.vdouble(0.0, 0.0, 0.0, 0.0)
+        ),
+    cms.PSet(
+        detector = cms.string("HCAL_ENDCAP"),
+        depths = cms.vint32(1, 2, 3, 4, 5, 6, 7),
+        seedingThreshold = cms.vdouble(0.1375, 0.275, 0.275, 0.275, 0.275, 0.275, 0.275),
+        seedingThresholdPt = cms.vdouble(0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0)
+        )
+)
+
+seedFinderThresholdsByDetector2019 = particleFlowClusterHBHE.seedFinder.thresholdsByDetector
+seedFinderThresholdsByDetector2019 = cms.VPSet(
```

Comment:
all off the 2019 values have to be the same as 2018.
Please reuse 2018 values in all relevant cases